cordova-plugin-appcenter-push/src/android/AppCenterPush.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:+'
+        classpath 'com.android.tools.build:gradle:3.+'
```

Comment:
Good catch, we actually have a policy of not using `+` in other repositories and that was missed here.
I would prefer an exact version instead (alt enter and resolve the warning in studio).

Reply:
Ah, you got it.